python/lsst/sims/maf/plots/plotHandler.py

```diff
@@ -9,6 +9,21 @@
 
 __all__ = ['PlotHandler', 'BasePlotter']
 
+
+def none_min(x):
+    if None in x:
```

Comment:
I am just impossible to please...

What if the lims specified by hand are `None`, but the lims from `plt.xlim()` and 'plt.ylim()` are not `None`?  Wouldn't we want to return the non-None values?

I am going to ping @rhianonlynne on this, since she originally wrote the code and probably knows what it is supposed to do.